catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/AccessControlPreIngestPluginTest.java

```diff
@@ -110,20 +110,6 @@ public void testUpdateIgnoreNonAclMetacards() {
     assertThat(next.getAttribute(Core.METACARD_OWNER), nullValue());
   }
 
-  //  @Test
```

Comment:
remove?

Reply:
We don't do these sorts of updates in the ACL pre-ingest plugin anymore, so the test case didn't make sense.